cms/templates/partials/admissions-section.html

```diff
@@ -0,0 +1,61 @@
+{% load image_version_url wagtailcore_tags humanize %}
+<div class="container-fluid admissions-section">
+    <div class="row">
+        <img class="admissions-image" src="{% image_version_url page.admissions_image 'fill-1920x540' %}" alt="Admissions" />
+    </div>
+    <div class="row mb-4">
+        <div class="col-md-10 offset-md-2 col-sm-12 offset-sm-0 pt-5 bg-white">
+            <div class="container-fluid admissions-section">
+                <div class="row">
+                    <div class="col-md-4 col-sm-12 mb-2">
+                        <h1 class="section-heading">How to Apply</h1>
+                        <h2 class="admissions-heading mt-2"><a href="/admissions">Admissions</a></h2>
+                    </div>
+                    <div class="col-md-7 col-sm-12 admissions-detail">
+                        <div class="container-fluid">
+                            <div class="row">
+                                <div class="col">
+                                    {{ page.details|richtext }}
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col d-flex flex-row">
+                                    <div class="flex-shrink-0">
+                                        <a href="/admissions" class="btn btn-primary btn-apply">Apply Now</a>
```

Comment:
Can you add a `@property` to the bootcamp run page class for this application URL? We don't yet know what this link will be exactly, but it won't go to `/admissions`. You can just return `/admissions` in that property method as a placeholder though

Reply:
Added